DOCS-725 - Prepping for Big Animal -> Cloud Service (#5750)
EnterpriseDB/docs: advocacy_docs/edb-postgres-ai/console/estate/reference/metrics_ingestion.mdx

---
title: Metrics Ingestion into the EDB Postgres AI Estate
navTitle: Metrics Ingestion
description: The metrics that can be ingested into the EDB Postgres AI Estate.
---

> All agents in the EDB Postgres AI Estate can send metrics to the Estate control plane.

Reviewer: Do we want to talk about the CP?

Author: For now, Control Plane is sufficiently technical to refer to the abstract entitity that is the Estate infra.

Suggestion for if we go with Platform/Etate
```suggestion
All agents that you configure in your Estate can send metrics to the EDB Postgres AI Platform.
```

> Currently the following metrics are ingested into the Estate control plane:

Reviewer: This list applies to bare metal instances only... although you did say 'all agents' so that's kind of implied. Although actually the bare metal agent also sends some data about the host OS although it's not exposed anywhere in UI.
